geo/src/algorithm/mod.rs

```diff
@@ -263,8 +263,19 @@ pub use triangulate_earcut::TriangulateEarcut;
 
 /// Triangulate polygons using an (un)constrained [Delaunay Triangulation](https://en.wikipedia.org/wiki/Delaunay_triangulation) algorithm.
 #[cfg(feature = "spade")]
+pub mod triangulate_delaunay;
+#[cfg(feature = "spade")]
```

Comment:
Should we get a more meaningful feature name, like `triangulate_delaunay`?

Reply:
To be clear, I don't feel strongly about this part, and am happy to leave it as is.

Reply:
I'd like to keep `spade` for now, since we'll hopefully also be making use of some other `spade` functionality (Voronoi diagrams) in the near future.